### PR TITLE
CCD 770

### PR DESCRIPTION
**Pull request by hbrown-ST**
Reccat needed a new base rmap for niriss_mask_nrm.

### CODE REVIEW

File: test/core/test_reftypes.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ def test_reftypes_hst_get_filekinds(default_shared_state):
...
+    expected_types = ['abvegaoffset', 'all', 'amplifier', 'apcorr', 'area', 'dark', 'distortion', 'drizpars', 'extract1d', 'filteroffset', 'flat', 'gain', 'ipc', 'linearity', 'mask', 'mask-nrm', 'pars-chargemigrationstep', 'pars-darkpipeline', 'pars-detector1pipeline', 'pars-image2pipeline', 'pars-jumpstep', 'pars-outlierdetectionstep', 'pars-rampfitstep', 'pars-sourcecatalogstep', 'pars-spec2pipeline', 'pars-tweakregstep', 'pars-undersamplecorrectionstep', 'pathloss', 'persat', 'photom', 'readnoise', 'regions', 'saturation', 'speckernel', 'specprofile', 'spectrace', 'specwcs', 'superbias', 'throughput', 'trapdensity', 'trappars', 'wavelengthrange', 'wavemap', 'wcsregions', 'wfssbkg']
```
Comment (stscieisenhamer) [inline test/core/test_reftypes.py:214]: Should "mask-nrm" be just "nrm"? "nrm" is the new reftype, no?

**Review by stscieisenhamer — CHANGES_REQUESTED** [review]

File: test/core/test_reftypes.py (hunk 1 above)
Comment (hbrown-ST) [reply test/core/test_reftypes.py:214]: Thanks, this was still from my previous iteration. Fixing.

**Review by stscieisenhamer — APPROVED** [review]
As the ci failures are unrelated and unexplained, approving this particular PR. However, it would be good to build a dev container and deploy to both jwst dev and hst dev and run the tests locally there before making a release.

**Comment by hbrown-ST** [comment]
Was able to deploy changes to both HST and JWST dev machines and tests passed fine on each. The CI errors must be due to something else.

**Pull request merged**